[2022-09-01 22:22:43,455] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: trigger_sensor.waiting_for_run_file manual__2022-09-01T22:22:42.537139+00:00 [queued]>
[2022-09-01 22:22:43,512] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: trigger_sensor.waiting_for_run_file manual__2022-09-01T22:22:42.537139+00:00 [queued]>
[2022-09-01 22:22:43,516] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-01 22:22:43,522] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-09-01 22:22:43,523] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-01 22:22:43,574] {taskinstance.py:1397} INFO - Executing <Task(FileSensor): waiting_for_run_file> on 2022-09-01 22:22:42.537139+00:00
[2022-09-01 22:22:43,586] {standard_task_runner.py:52} INFO - Started process 94999 to run task
[2022-09-01 22:22:43,599] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'trigger_sensor', 'waiting_for_run_file', 'manual__2022-09-01T22:22:42.537139+00:00', '--job-id', '3728', '--raw', '--subdir', 'DAGS_FOLDER/trigger_dag.py', '--cfg-path', '/tmp/tmpn_jt347g', '--error-file', '/tmp/tmpbvdy7xad']
[2022-09-01 22:22:43,609] {standard_task_runner.py:80} INFO - Job 3728: Subtask waiting_for_run_file
[2022-09-01 22:22:43,793] {task_command.py:371} INFO - Running <TaskInstance: trigger_sensor.waiting_for_run_file manual__2022-09-01T22:22:42.537139+00:00 [running]> on host 9d5d2a77e6ba
[2022-09-01 22:22:44,026] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=trigger_sensor
AIRFLOW_CTX_TASK_ID=waiting_for_run_file
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T22:22:42.537139+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-01T22:22:42.537139+00:00
[2022-09-01 22:22:44,048] {base.py:68} INFO - Using connection ID 'my_file_system_practice' for task execution.
[2022-09-01 22:22:44,050] {filesystem.py:58} INFO - Poking for file /opt/***/dags/test.txt
[2022-09-01 22:22:54,058] {base.py:68} INFO - Using connection ID 'my_file_system_practice' for task execution.
[2022-09-01 22:22:54,059] {filesystem.py:58} INFO - Poking for file /opt/***/dags/test.txt
[2022-09-01 22:23:04,078] {base.py:68} INFO - Using connection ID 'my_file_system_practice' for task execution.
[2022-09-01 22:23:04,079] {filesystem.py:58} INFO - Poking for file /opt/***/dags/test.txt
[2022-09-01 22:23:04,081] {filesystem.py:63} INFO - Found File /opt/***/dags/test.txt last modified: 20220901222301
[2022-09-01 22:23:04,082] {base.py:301} INFO - Success criteria met. Exiting.
[2022-09-01 22:23:04,102] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=trigger_sensor, task_id=waiting_for_run_file, execution_date=20220901T222242, start_date=20220901T222243, end_date=20220901T222304
[2022-09-01 22:23:04,149] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-01 22:23:04,235] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
